source/amazon/installation.rst

```diff
@@ -155,78 +160,45 @@ Plugin configuration
 
     # vi /var/ossec/etc/ossec.conf
 
-2. Add the following block of configuration to enable the integration, enter the AWS IAM User credentials you created before:
+2. Add the following block of configuration to enable the integration, enter the AWS IAM User credentials you created before and the AWS Account ID of the CloudTrail logs to be processed:
 
 .. code-block:: xml
 
+
     <wodle name="aws-cloudtrail">
       <disabled>no</disabled>
-      <bucket>wazuh-cloudtrail</bucket>
-      <access_key>insert_access_key</access_key>
-      <secret_key>insert_secret_key</secret_key>
-      <remove_from_bucket>no</remove_from_bucket>
       <interval>10m</interval>
       <run_on_start>no</run_on_start>
+      <error_not_skip><error_not_skip>
+      <cloudtrail>
+        <bucket>wazuh-cloudtrail</bucket>
+        <access_key>insert_access_key</access_key>
+        <secret_key>insert_secret_key</secret_key>
+        <aws_account_id>insert_account_id</aws_account_id>
+      </cloudtrail>
     </wodle>
 
+To monitor CloudTrail logs for multiple AWS accounts, configure multiple <cloudtrail> options within the aws-cloudtrail wodle
```

Comment:
You've added support for _Support IAM credential profiles_ and _IAM roles_. I'm missing a section in our documentation explaining how to configure that. I'd do it explaining that two kinds of authentication are allowed: using an access/secret key or using IAM credential profiles and roles.

Reply:
This makes sense.  I can expand on the authentication options and provide examples...